spec.md

```diff
@@ -1821,6 +1821,15 @@ message ControllerExpandVolumeRequest {
   // Secrets required by the plugin for expanding the volume.
   // This field is OPTIONAL.
   map<string, string> secrets = 3 [(csi_secret) = true];
+
+  // Volume capability describing how the CO intends to use this volume.
+  // This allows SP to determine if volume is being used as a block
+  // device or mounted file system. For example - if volume is
+  // being used as a block device - the SP MAY choose to skip calling
+  // NodeExpandVolume on the node by setting node_expansion_required
+  // to false in ControllerExpandVolumeResponse.
+  // This is an OPTIONAL field.
+  VolumeCapability volume_capability = 4;
```

Comment:
A k8s PV may have multiple access modes. How do we map that to a single VolumeCapability?

Reply:
This was discussed in CSI meeting. Resizing is only dependent on access_type and not on access_mode when it comes to whether volume can be resized(how) and hence just having singular capability should be enough.

Reply:
Should we say that in the spec here. That only the `access_type` will be set not `access_mode`?

Reply:
The way `VolumeCapability` is defined `access_type` and `access_mode` both will be always set. It is another thing that expansion calls don't care about access_mode but there is just no way to instantiate `VolumeCapability` and not have `access_type` set on it.

Reply:
Added additional note.